db/migrate/20190404135622_appointments.rb

```diff
@@ -14,7 +14,7 @@ def change
     create_table :appointments do |t|
       t.date :date
       t.integer :item_number
-      t.string :status, default: 'Booked'
+      t.string :status, default: 'booked'
```

Comment:
When a migration has been merged you can't do any modifications to it. as it was already run on other machines. So you need to make a new one with the updates needed.